modules/doc/content/applications/thermal_hydraulics.md

```diff
@@ -0,0 +1,132 @@
+# Thermal Hydraulics Applications
+
+MOOSE provides versatile, general-purpose thermal-hydraulics applications. These applications solve for mass, momentum, and energy conservation in multicomponent, multiphase flows using incompressible, weakly-compressible, or fully compressible formulations for steady-state or transients in lumped parameters and/or multidimensional (1, 2, or full 3D) geometries.
+
+These applications are developed as part of the
+[Nuclear Energy Advanced Modeling and Simulation (NEAMS) program](https://neams.inl.gov/).
+
+!media thermal_hydraulics/misc/TH_scales_new.png
+       style=width:50%;display:block;margin-left:auto;margin-right:auto;
+       caption=MOOSE modules support from Reynold-Average Navier Stokes (RANS) Computational Fluid Dynamics (CFD) modeling to 0D lumped parameters modeling.
+       id=TH_Scales
+
+## MOOSE-based Modules for Thermal-Hydraulics Modeling
+
+| Module                                                                  | Scale                                  | Flow-Formulation                                                                                                                                             | Dimension                                                          | Typical Element Count | Typical Runtime | Typical Simulations                                                                                                                                                                                    |
+| :---------------------------------------------------------------------- | :------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------- | :-------------------- | :-------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Navier-Stokes Modue](navier_stokes/index.md)                           | Coarse-Mesh CFD \\ \\ RANS simulations | Incompressible, Weakly-Compressible, or Fully-Compressible\\ \\ Single- or Multi-Phase \\ \\ Single- or Multi-Component Flow                                 | Typically, 2D, 2D axisymmetric, or 3D \\ \\ Can also be used in 1D | 10,000                | 1 minute        | Flow through nuclear reactor core or plena \\ \\ 3D multi-phase flow in pipes \\ \\ Natural convection flow in open cavities                                                                           |
```

Comment:
```suggestion
| [Navier-Stokes Module](navier_stokes/index.md)                           | Coarse-Mesh CFD \\ \\ RANS simulations | Incompressible, Weakly-Compressible, or Fully-Compressible\\ \\ Single- or Multi-Phase \\ \\ Single- or Multi-Component Flow                                 | Typically, 2D, 2D axisymmetric, or 3D \\ \\ Can also be used in 1D | 10,000                | 1 minute        | Flow through nuclear reactor core or plena \\ \\ 3D multi-phase flow in pipes \\ \\ Natural convection flow in open cavities                                                                           |
```

Reply:
the run time and elements could be split ?
I dont like saying we do 1 minute runs with 10k elements
maybe we can have
10 k.                   1 minute
>100k                Setup-dependent